lib/chitragupta/cls.js

```diff
@@ -0,0 +1,26 @@
+const CLS = require('cls-hooked');
+const CONSTANTS = require('./constants');
+
+const NS = CLS.createNamespace(CONSTANTS.CHITRAGUPTA_CLS_ID);
+
+function get(key) {
+  let returnVal = null;
+  if (NS && NS.active) {
```

Comment:
should we activate NS if its not active?

Reply:
It is set to active when the function is called from the scope of `NS.run()`. So, we need not activate it otherwise.